agent/exploits/cve_2024_11972.py

```diff
@@ -0,0 +1,43 @@
+"""Agent Asteroid implementation for CVE-2024-11972"""
+
+import re
+
+from packaging import version
+
+from agent import definitions
+from agent import exploits_registry
+from agent.exploits import webexploit
+
+VERSION_PATTERN = re.compile(r"Stable tag:\s(\d+\.\d+(?:\.\d+)?)")
+MAX_VULNERABLE_VERSION = version.parse("1.8.9")
+VULNERABILITY_TITLE = (
+    "Hunk Companion Plugin Unauthenticated Arbitrary Plugin Installation"
+)
+VULNERABILITY_REFERENCE = "CVE-2024-11972"
+VULNERABILITY_DESCRIPTION = (
+    "The Hunk Companion plugin for WordPress is vulnerable to an issue that allows "
+    "unauthenticated POST requests to install and activate plugins directly from the WordPress.org repository. "
+    "Versions below 1.9.0 are affected. Attackers could use this flaw to introduce arbitrary functionality "
+    "to a WordPress instance, potentially leading to severe consequences."
+)
+RISK_RATING = "CRITICAL"
+
+
+@exploits_registry.register
+class CVE202411972Exploit(webexploit.WebExploit):
+    accept_request = definitions.Request(
+        method="GET", path="/wp-content/plugins/hunk-companion/readme.txt"
+    )
+    check_request = definitions.Request(
+        method="GET", path="/wp-content/plugins/hunk-companion/readme.txt"
+    )
```

Comment:
what is the difference between these two requests ?

Reply:
nothing :) , the webexploit.WebExploit which is the base class use these paths to get a response and then check for accept_pattern and version_pattern